src/routes/backoffice.ts

```diff
@@ -67,6 +67,51 @@ export const triggerBookingsWebhook = async (solarisAccountId) => {
   await triggerWebhook(TransactionWebhookEvent.BOOKING, payload);
 };
 
+const triggerAccountPKontoWebhook = async (person) => {
+  const { iban, id: accountId, is_pkonto: isPkonto } = person.account;
+
+  const payload = {
+    account_id: accountId,
+    person_id: person.id,
+    business_id: null,
+    updated_at: new Date().toISOString(),
+    is_pkonto: isPkonto,
+    iban,
+  };
+
+  await triggerWebhook(AccountWebhookEvent.ACCOUNT_PKONTO_CHANGE, payload);
+};
+
+export const setAccountIsPkontoFlag = async (personId, isPkonto) => {
+  const person = await getPerson(personId);
+
+  if (!person.account) return null;
+
+  const previousIsPkonto = person.account.is_pkonto;
+
+  if (isPkonto === "true") {
+    isPkonto = true;
+  } else if (isPkonto === "false") {
+    isPkonto = false;
+  }
+
+  person.account = {
+    ...person.account,
+    is_pkonto: isPkonto,
+  };
+
+  await savePerson(person);
+
+  if (isPkonto !== previousIsPkonto) {
+    await triggerAccountPKontoWebhook(person);
+  }
+};
+
+export const setAccountIsPkontoFlagHandler = async (req, res) => {
+  await setAccountIsPkontoFlag(req.params.personId, req.body.isPkonto);
```

Comment:
Meeting note: This will rather become setting of some values rather than a flag.